Fixes #10747 adding a shortcut to expand class definition. Thanks Denis.

Ducasse opened the pull request.

Ducasse committed:
Fixes #10747 adding a shortcut to expand class definition. Thanks Denis.

dionisiydk commented on src/Calypso-SystemTools-Core/ClyExpandClassDefinitionCommand.class.st:
```diff
@@ -5,9 +5,10 @@ Class {
 }
 
 { #category : #activation }
-ClyExpandClassDefinitionCommand class >> sourceCodeMenuActivation [
```
Do you want to remove command from the menu?

Ducasse commented:
No but it did not remove it when I remove it. I will add it back.

Ducasse committed:
Add back the method 

sourceCodeMenuActivation
   <classAnnotation>
   ^SycSourceCodeMenuActivation byRootGroupItemOrder: 1.2 for: ClyClassDefinitionContext

because it should define a menu (now removing it did not remove the menu).
